tests/kubeTest.sh

```diff
@@ -47,6 +47,7 @@ done
 [[ -z ${NAMESPACE} ]] && NAMESPACE="$(generate_namespace)"
 
 if [[ -z ${ISTIOCLI} ]]; then
+    echo "Downloading istioctl from ${ISTIOCTL_URL}/istioctl-linux"
     wget -q -O "${TEST_DIR}/istioctl" "${ISTIOCTL_URL}/istioctl-linux" || error_exit "Could not download istioctl"
```

Comment:
though maybe there is a way to have stderr ... maybe removing the -q would be better